### PR TITLE
Add SKR PRO v1.1 config

### PR DESCRIPTION
**Pull request by vumario**
Added basic BTT SKR PRO v1.1 map file.

Some peripherals would require SPI and I2C driver changes because of some different pin combinations used so I left them out for now, but everything that is included is tested working.

This PR depends on my previous PR to support all 6 axes, so that one should probably get merged before this.

br,
Mario

### CODE REVIEW

**Comment by vumario** [comment]
I made some more changes to support I2C on this board.

I don't really know if this is the way to do it, but it does work. If You have any comments or ideas, let me know.

File: Inc/driver.h [hunk 1]
```diff
@@ -202,9 +204,16 @@
...
+#ifndef SPI_PORT
+#define SPI_PORT 1
+#endif
+
 #if EEPROM_ENABLE|| KEYPAD_ENABLE || (TRINAMIC_ENABLE && TRINAMIC_I2C)
   #if defined(NUCLEO_F411) || defined(NUCLEO_F446)
     #define I2C_PORT 1 // GPIOB, SCL_PIN = 8, SDA_PIN = 9
+  #elif defined(BOARD_SKR_PRO_1_1)
+    #define I2C1_ALT_PINMAP
+    #define I2C_PORT 1 // GPIOB, SCL_PIN = 6, SDA_PIN = 7
```
Comment (terjeio) [inline Inc/driver.h:216]: IMO these lines should be moved to the SKR board map and the code in this file changed to allow that.

**Comment by terjeio** [comment]
This PR looks good, I need a bit more time for review before merging. I hope that is ok with you.

File: Inc/driver.h (hunk 1 above)
Comment (vumario) [reply Inc/driver.h]: OK, I made that change

**Comment by terjeio** [comment]
You have to merge the latest version befor updating the PR?

**Comment by vumario** [comment]
Yeah, I wanted to do that, but ended up making a mess... At this point I think it would easier to just make a new PR

**Pull request closed without merging**